_scripts/build_npm.ts

```diff
@@ -14,8 +14,21 @@ await build({
     "./mod.ts",
     "./src/environment.ts",
     "./src/loader.ts",
+    "./src/url_loader.ts",
     "./src/tokenizer.ts",
-    "./plugins/auto_trim.ts",
```

Comment:
Don't remove this plugin from the exports.

Reply:
Problem solved! @oscarotero